packages/millicast-sdk/src/index.js

```diff
@@ -4,12 +4,13 @@ import Signaling from './Signaling'
 import Director from './Director'
 import Publish from './Publish'
 import View from './View'
-
+import { DOLBY_SEI_TIMESTAMP_UUID } from './utils/Codecs'
```

Comment:
```suggestion

```
If it is not going to be used in any demo app do not know if it would be recommended to export it. As it is mainly a UUID inserted by the server and the SDK uses it for internal checks